pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -462,6 +462,42 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 			newMounterErr)
 	}
 
+	deviceMountPath := ""
+	if attachablePlugin != nil {
```

Comment:
Could you add a comment for this part of checking?

Reply:
Also move this part above the defination of volumeMounter?

Reply:
Ack